adhocteam/Head-Start-TTADP
# Multiselect's value is an array of strings

**Description of change**
The multiselect component used an array of objects for the value with the shape [{ label: ..., value: ... }]. This PR updates the value to be an array of strings.

Add testing library linter and fix all the violations.

I had a good fight getting the new linter to work in VS code. If you find eslint inside vs code no longer works after pulling this change down try to set

```
    "eslint.workingDirectories": [
        ".", "./frontend"
    ],
```

in your vscode settings.json file.

**How to test**

1) Pull down code, install dependencies
2) Fill out an activity report, make sure your console is open
3) Click submit, in your console any multiselect field should have a value like `["one", "two", ...]`, while previously the value would be `[{label: "one", value: "one"}, ...]`

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/98

**Checklist**

- [n/a] Meets issue criteria
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
